src/app/checkboxes/checkboxes.demo.html

```diff
@@ -49,21 +49,55 @@
             <label>Indeterminate Checkbox</label>
             <div class="checkbox-inline">
                 <clr-checkbox type="checkbox" id="checkbox_9" [(clrIndeterminate)]="indeterminateState">
-                    Indeterminate Checkbox.
+                    Clarity indeterminate checkbox
                 </clr-checkbox>
                 <div>
-                    <div>
-                        <b>NOTE:</b> Native checkbox elements must set indeterminate state with JavaScript.
-                    </div>
                     <button class="btn btn-sm" (click)="onToggleIndeterminateState($event)" type="button">
                         Toggle Indeterminate State
                     </button>
                 </div>
+                <div class="checkbox-inline">
+                    <input type="checkbox"
+                           id="checkrads_10"
+                           [indeterminate]="nativeIndeterminateState">
+                    <label for="checkrads_10">Native indeterminate checkbox</label>
+                    <div>
+                        <b>NOTE:</b> Native checkbox elements must set indeterminate state with JavaScript.
+                    </div>
+                </div>
             </div>
         </div>
     </section>
 </form>
 
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <clr-checkbox
+                    name="termsCheckbox"
+                    class="checkbox-inline"
```

Comment:
You don't need to manually add the class since you already have `[clrInline]="true"`.